### ISSUE TITLE
Full Episodes Broken: "There are no full episode shows to list"

### ISSUE THREAD

**Issue by jpborjas** [plexinc-plugins/HGTV.bundle#19]
This was working just a few days ago. I guess the format changed? There are shows in the main URL http://www.hgtv.com/shows/full-episodes , but the list is coming back as empty.

**Comment by Sardonimous** [comment]
Changing this line

  for item in HTML.ElementFromURL(FULLEP_URL).xpath('//div[@class="parbase editorialPromo section"]//ul/li'):
to
  for item in HTML.ElementFromURL(FULLEP_URL).xpath('//div[@class="editorialPromo parbase section"]//ul/li'):

fixed this for me...

**Comment by sander1** [comment]
Thanks for the info! The code has been updated and an update has just been released through the Channel Directory.

**Comment by jpborjas** [comment]
Interesting, I upgraded via the GUI, and my line looks like this:

    for item in HTML.ElementFromURL(FULLEP_URL).xpath('//div[contains(@class, "editorialPromo")]//ul/li'):

VERSION:
57debef (Mon Nov 16 14:29:41 UTC 2015)

I deleted the channel, witch deleted the bundle file, reinstalled it, and the line still looks as above. 

I found in the source for the site, the class listed above "editorialPromo parbase section", so even though my code didn't look as the code before the changes, I changed the part in quotes, to no avail. 

I then went to the source here on this site, for the _init_.py file, and the line listed above doesn't exist. This is what the current code, updated 3 hours ago, shows:

     for item in HTML.ElementFromURL(FULLEP_URL).xpath('//div[contains(@class, "editorialPromo")]//ul/li'):

What am I missing?
